src/types.ts

```diff
@@ -15,6 +15,17 @@ export interface ImportCommon {
   priority?: number
   /** If this import is disabled */
   disabled?: boolean
+  /**
+   * Metadata of the import
+   */
+  meta?: {
+    /** Short description of the import */
+    description?: string
+    /** URL to the documentation */
+    docsUrl?: string
```

Comment:
would we want metadata for presets (e.g. single docs url for entire bundle that gets inherited by individual imports?)

Reply:
Actually it's already inherited :)

Reply:
Perfect!